src/client/results/mostfrequent/users/UserItemsList.js

```diff
@@ -70,4 +76,20 @@ let UserItem = ({ dispatch, userInfo }) => (
 );
 UserItem = connect()(UserItem);
 
+const UserItemMostUsedWords = ({ usersMostUsedWords }) => (
+  <div className="ui pointing fluid dropdown">
+    <div className="text"><b>{usersMostUsedWords[0].word}</b> x{usersMostUsedWords[0].count}</div>
```

Comment:
Semantics - \<strong> says "this is an important element" whereas \<b> says "this is a bold element" - the point being that HTML should be declarative, and not actually mention styling in any of the tags.

https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em